cime_config/testmods_dirs/atmlndactive/rtm_off/user_nl_mosart

```diff
@@ -0,0 +1,6 @@
+! disable rtm to allow short ERS and ERP run to complete. 
+! If rtm active, due to large rtm time step, restart pointer for ROF
+! is always ahead of that for atm at the end of the first run. And in 
+! restart run, the restart point for ROF becomes later than the run stop time
+
+ do_rtm =.false.
```

Comment:
To get past this limitation (missing rpointer for rtm), I used to enforce RTM=NULL in env_build.xml. I think that approach doesn't even build RTM, so we may save some time there. I am not sure how `do_rtm=.false.` is implemented. If it is cleaner to use `do_rtm=.false`, we should stick with this way.

Reply:
Good point, Balwinder. Both would work for the sake of disabling mosart. It may still be useful to build rtm.

Reply:
Balwinder, looks like RTM_MODE (taking value ACTIVE or NULL) is no more supported. It was used when we have component/rtm. I think mosart introduced this runtime option do_rtm to do the equivalent.

Reply:
Ok, That is good to know. Thanks, @wlin7 !